site2/docs/deploy-kubernetes.md

```diff
@@ -1,10 +1,39 @@
 ---
 id: deploy-kubernetes
-title: Deploy Pulsar on Kubernetes
+title: Overview
 sidebar_label: "Kubernetes"
 ---
 
-To get up and running with these charts as fast as possible, in a **non-production** use case, we provide
-a [quick start guide](getting-started-helm.md) for Proof of Concept (PoC) deployments.
+The Apache Pulsar [Helm chart](https://github.com/apache/pulsar-helm-chart) provides one of the most convenient ways to deploy and operate Pulsar on Kubernetes. With all the required components, the Helm chart is scalable and thus suitable for large-scale deployments.
 
-To configure and install a Pulsar cluster on Kubernetes for production usage, follow the complete [Installation Guide](helm-install.md).
\ No newline at end of file
+The Apache Pulsar Helm chart contains all components to support the features and functions that Pulsar delivers. You can install and configure these components separately.
+
+- Pulsar core components
+  - ZooKeeper
+  - Bookies
+  - Brokers
+  - Function workers
+  - Proxies
+- Control center
+  - Pulsar Manager
+  - Prometheus
+  - Grafana
+
+
+## What's Next?
+
+* To get up and running with Pulsar Helm chart as fast as possible for Proof of Concept (PoC) in a **non-production** use case, see the [quick start guide](getting-started-helm.md). 
```

Comment:
I suggest removing this "quick start guide" cause https://pulsar.apache.org/docs/getting-started-helm just shows a sample of how to install `pulsar-helm-chart` on minikube which is very similar to the formal installation https://pulsar.apache.org/docs/helm-deploy. I'd like to remove the "quick start" and just keep the formal installation.

Reply:
I think it's okay to leave the link here as a quick start option and it's up to the readers to the next move.